src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -81,6 +86,32 @@ const TableHeroImage = ({
         {subtitle &&
           <span className={styles.subText}>
             {subtitle}
+            <span className={styles.iconBox}>
+              {randomed &&
+                <span data-tip data-for="randomed">
+                  <IconDice fill="currentcolor" />
+                  <ReactTooltip id="randomed" place="top" type="light" effect="solid">
```

Comment:
I think @blukai added a way to do lighter-weight tooltips if you need text only.